src/core_plugins/kibana/public/discover/directives/index.js

```diff
@@ -25,8 +25,14 @@ import {
   DiscoverNoResults,
 } from './no_results';
 
+import {
```

Comment:
references to rollup message can be removed with implementation of `renderUnsupportedIndexPatternMessage` or similar